src/main/java/frc/robot/io/implementations/motor/MotorIOSparkMax.java

```diff
@@ -49,6 +53,13 @@ public MotorIOSparkMax(MotorIOBaseSettings motorSettings, SparkMaxSettings spark
         SparkBase.PersistMode.kPersistParameters);
     encoder = motor.getEncoder();
     pid = motor.getClosedLoopController();
+
+    if (motorSettings.reverseLimitChannel > -1) {
+      reverseLimit = new DigitalInput(motorSettings.reverseLimitChannel);
+    }
+    if (motorSettings.reverseLimitChannel > -1) {
+      forwardLimit = new DigitalInput(motorSettings.reverseLimitChannel);
```

Comment:
Looks like a copy/paste typo.  These should be forwardLimitChannel (or maxLimitChannel?)

```suggestion
    if (motorSettings.forwardLimitChannel > -1) {
      forwardLimit = new DigitalInput(motorSettings.forwardLimitChannel);
```

Reply:
Will fix.